From ec1ef5de8c87b5f6e3feea364ea2056070c244d0 Mon Sep 17 00:00:00 2001
From: David Lechner <david@lechnology.com>
Date: Tue, 28 Feb 2017 20:22:30 -0600
Subject: [PATCH 9/9] Make BeagleBone Blue battery LED labels match the circuit
 board

Change the BeagleBone Blue battery indicator LED device nodes so that the label matches what is actually printed on the circuit board. This makes identification more obvious.
---
 arch/arm/boot/dts/am335x-boneblue.dts | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/am335x-boneblue.dts b/arch/arm/boot/dts/am335x-boneblue.dts
index a6b8051b5bbc..c5017029b0d3 100644
--- a/arch/arm/boot/dts/am335x-boneblue.dts
+++ b/arch/arm/boot/dts/am335x-boneblue.dts
@@ -610,25 +610,25 @@
 		};
 
 		batt_1_led {
-			label = "bat0";
+			label = "bat25";
 			gpios = <&gpio0 27 GPIO_ACTIVE_HIGH>;
 			default-state = "off";
 		};
 
 		batt_2_led {
-			label = "bat1";
+			label = "bat50";
 			gpios = <&gpio0 11 GPIO_ACTIVE_HIGH>;
 			default-state = "off";
 		};
 
 		batt_3_led {
-			label = "bat2";
+			label = "bat75";
 			gpios = <&gpio1 29 GPIO_ACTIVE_HIGH>;
 			default-state = "off";
 		};
 
 		batt_4_led {
-			label = "bat3";
+			label = "bat100";
 			gpios = <&gpio0 26 GPIO_ACTIVE_HIGH>;
 			default-state = "off";
 		};
-- 
2.11.0

